HPC/README.md

```diff
@@ -184,6 +186,11 @@ Thread model: posix
 InstalledDir: /shared/arm/arm-linux-compiler-23.04_Ubuntu-20.04/bin
 ```
 
+### Storage
+Some HPC applications require significant amounts of file I/O, however HPC instance types (Graviton instances included) don't have significant amounts of local storage, and have extremely limited EBS bandwidth and IOPS.  Relying on EBS on each node can cause surprise slow-downs when the instance runs out of EBS burst credits.  This is one reason we don't recommend using an hpc7g (or any HPC instance type) for headnodes, since the headnode performs additional I/O as the scheduler.  For these reasons the following recommendations are made:
```

Comment:
Maybe we should say "limited EBS bandwidth" instead of "extremely limited" or something like "less EBS bandwidth than compute optimized instances" if that is actually true. Also the HPC7g doesn't have any local storage, right?